src/repository/forms.py

```diff
@@ -519,3 +519,28 @@ def save(self, commit=True):
             field.save()
 
         return field
+
+
+class PreprinttoArticleForm(forms.Form):
+    license = forms.ModelChoiceField(queryset=submission_models.Licence.objects.none())
+    section = forms.ModelChoiceField(queryset=submission_models.Section.objects.none())
+    stage = forms.ChoiceField(
+        choices=()
+    )
+    force = forms.BooleanField(
+        required=False,
+        help_text='If you want to force the creation of a new article object even if one exists, check this box.',
```

Comment:
Perhaps we can explain what happens to the old article here

Reply:
👍🏻